svelte-kit-scss/src/lib/components/shared/layouts/LayoutPageHeader.svelte

```diff
@@ -3,9 +3,20 @@
   import { onDestroy, onMount } from 'svelte';
   export let gray = false;
   export let paddingTop = false;
+  let stickyElement: HTMLDivElement;
+  let headerElement: HTMLDivElement;
+
+  let stickyElementTopPx: number | undefined;
+  let headerElementLeftPx: number | undefined;
+  let headerElementWidthPx: number | undefined;
 
   const handleScroll = () => {
-    // todo: handle sticky
+    const rectSticky = stickyElement?.getBoundingClientRect();
+    stickyElementTopPx = rectSticky?.top;
+
+    const rectHeader = headerElement?.getBoundingClientRect();
+    headerElementLeftPx = rectHeader?.left;
+    headerElementWidthPx = rectHeader?.width;
```

Comment:
Maybe we can write this as: 

```const handleScroll = () => {
    if (stickyElement) {
      const rectSticky = stickyElement.getBoundingClientRect();
      stickyElementTopPx = rectSticky.top;
    }

    if (headerElement) {
      const rectHeader = headerElement.getBoundingClientRect();
      headerElementLeftPx = rectHeader.left;
      headerElementWidthPx = rectHeader.width;
    }
  };
```
  In order not to have all that question marks around

Reply:
got it!